models/models.js

```diff
@@ -25,3 +25,28 @@ exports.selectReviewById = (review_id) => {
       return Promise.reject(err);
     });
 };
+
+exports.updateVotes = (review_id, voteShift) => {
+  const { inc_votes } = voteShift;
+  return db
+    .query(
+      `UPDATE reviews SET votes = votes + $2 WHERE review_id = $1 RETURNING *;`,
+      [review_id, inc_votes]
+    )
+    .then(({ rows: review }) => {
+      if (!review.length) {
+        return Promise.reject({ status: 404, msg: "ID not found" });
+      }
+      if (review[0].votes < 0) {
+        const votes = review[0].votes - inc_votes;
```

Comment:
this error handling doesn't seem like enough for your limitation. By the time you've checked if it's negative, the damage has already been done and your table has a negative value in it. You'll want to stop the UPDATE from happening.